src/Applications/LDAS_App/ldas_setup

```diff
@@ -1456,7 +1456,7 @@ def _printExeInputKeys(rqdExeInpKeys):
     print '#  (3) MET_TAG  must be set to [ADAS_EXPID]__Nx+-          #'
     print '#      MET_PATH must be set as follows for                 #'
     print '#        LADAS_COUPLING = 1:                               #'        
-    print '#          [ADAS_EXPDIR]/lana/forc                         #'        
+    print '#          ../../scratch                                   #'        
```

Comment:
I suppose the suggested relative path `../../scratch` assumes that `./ldas_setup` is run after `cd` to `./[EXP_ID]/install/bin/`.  I'm not sure if `ldas_setup` could be run from elsewhere (by supplying the full path to `ldas_setup`).  If so, then the user might end up running `ldas_setup` from a different "_relative_" location.

It would probably be safer (and in any case clearer) to replace `../../scratch` with `[LDAS_EXPDIR]/scratch/` here, or perhaps use `[full_path]/[LDAS_EXPID]/scratch/` (similar to how we defined `[ADAS_EXPDIR]` in item `(0)` above).  Does that make sense?